core/templates/dev/head/pages/exploration_editor/translation_tab/TranslatorOverviewDirective.js

```diff
@@ -56,11 +97,30 @@ oppia.directive('translatorOverview', [
                     languageCode))
               };
             }));
+          };
+
+          var updateMode = function() {
```

Comment:
This doesn't seem to be updating the mode? I don't understand....

Reply:
Oh you mean update the $scope values. Hmmm.. The name is misleading. Can you think of a better name?